proto/drand_pb.proto

```
@@ -0,0 +1,8 @@
+syntax = "proto3";
+
+package drand_pb;
```

Buf (1.72.0)

[error] Files with package "drand_pb" must be within a directory "drand_pb" relative to root but were in directory "proto".

(PACKAGE_DIRECTORY_MATCH)